EditorExtensions/Settings/WESettings.cs

```diff
@@ -369,6 +369,12 @@ public sealed class JavaScriptSettings : LinterSettings<JavaScriptSettings>, IMi
         [Description("Forces all rules to be terminated with semicolons when true.")]
         [DefaultValue(true)]
         public bool TermSemicolons { get; set; }
+
+		[Category("Minification")]
+		[DisplayName("Custom base url for resources")]
+		[Description("Specifies a custom base url in case your application is not running directly on the root directory.")]
```

Comment:
> As for the category, I've used Minification because one could bundle the files and not minify them and as a consequence not have the same issue.

On the contrary, IMO the `GenerateSourceMaps` should also under `Bundle` category, because (if I remember it correctly) **`.js.map`** file is generated regardless of minification setting. In case minification is enabled, it *additionally* generates a **`.min.js.map`**.